server/meshmodel/aws-applicationautoscaling-controller/relationships/edge_permission.json

```diff
@@ -0,0 +1,56 @@
+{
+  "apiVersion": "core.meshery.io/v1alpha1",
+  "kind": "Edge",
+  "metadata": {
+    "description": "AWS Application Auto Scaling allows you to automatically adjust the capacity of your AWS resources, such as EC2 instances, DynamoDB tables, and ECS services, in response to demand or defined schedules. When integrating AWS Application Auto Scaling with IAM roles, you empower your scalable resources to interact with other AWS services securely and efficiently.for example -Autoscaling of kind AdoptedResource needs  an edge premission to roll out instances"
+  },
+  "model": {
+    "name": "aws-applicationautoscaling-controller",
+    "version": "1.0.13",
+    "displayName": "AWS Applilcation Autoscaling ",
+    "category": {
+      "name": "Provisioning",
+      "metadata": null
+    },
+
+    "metadata": {}
+  },
+  "subType": "Permission",
+  "rego_query": "edge_binding_relationship",
+  
+  "selectors": {
+
+    "allow": {
+
+      "from": [
+
+        {
+          "kind": "AdoptedResource",
+          "model": "aws-applicationautoscaling-controller",
+          "match": {
+
+            "self": [],
```

Comment:
Please add the appropriate path in the 'self' field under 'match'.

Reply:
@deepak4566 have you addressed this feedback?